src/daemon/daemon.cpp

```diff
@@ -1563,18 +1563,8 @@ try // clang-format on
 
         for (const auto& name : instances_to_suspend)
```

Comment:
Might as well replace the loop with a call to cmd_vms...

Reply:
Sure, this is a good candidate to switch over to that.

Reply:
Ok, I updated this to use `cmd_vms()`.